os/wqueue/Make.defs

```diff
@@ -0,0 +1,62 @@
+###########################################################################
+#
+# Copyright 2016 Samsung Electronics All Rights Reserved.
```

Comment:
Please check the license year. I know that this file is just moved, but I'm not sure we can keep the previous year.